datasets/bsd_ja_en/bsd_ja_en.py

```diff
@@ -0,0 +1,164 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Japanese-English Business Scene Dialogue (BSD) dataset. """
+
+from __future__ import absolute_import, division, print_function
+
+import json
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{rikters-etal-2019-designing,
+    title = "Designing the Business Conversation Corpus",
+    author = "Rikters, Matīss  and
+      Ri, Ryokan  and
+      Li, Tong  and
+      Nakazawa, Toshiaki",
+    booktitle = "Proceedings of the 6th Workshop on Asian Translation",
+    month = nov,
+    year = "2019",
+    address = "Hong Kong, China",
+    publisher = "Association for Computational Linguistics",
+    url = "https://www.aclweb.org/anthology/D19-5204",
+    doi = "10.18653/v1/D19-5204",
+    pages = "54--61"
+}
+"""
+
+
+_DESCRIPTION = """\
+This is the Business Scene Dialogue (BSD) dataset,
+a Japanese-English parallel corpus containing written conversations
+in various business scenarios.
+
+The dataset was constructed in 3 steps:
+  1) selecting business scenes,
+  2) writing monolingual conversation scenarios according to the selected scenes, and
+  3) translating the scenarios into the other language.
+
+Half of the monolingual scenarios were written in Japanese
+and the other half were written in English.
+
+Fields:
+- id: dialogue identifier
+- no: sentence pair number within a dialogue
+- en_speaker: speaker name in English
+- ja_speaker: speaker name in Japanese
+- en_sentence: sentence in English
+- ja_sentence: sentence in Japanese
+- original_language: language in which monolingual scenario was written
+- tag: scenario
+- title: scenario title
+"""
+
+_HOMEPAGE = "https://github.com/tsuruoka-lab/BSD"
+
+_LICENSE = "CC BY-NC-SA 4.0"
+
+_REPO = "https://raw.githubusercontent.com/tsuruoka-lab/BSD/master/"
+
+_URLs = {
+    "train": _REPO + "train.json",
+    "dev": _REPO + "dev.json",
+    "test": _REPO + "test.json",
+}
+
+
+class Bsd(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class BsdJaEn(datasets.GeneratorBasedBuilder):
```
this must be the camel vase version of the dataset script name `bsd_ja_en`